test.py

```diff
@@ -1,34 +1,4 @@
-import time
-import math
-import cv2
+from run import PipelineRunner
 
-import vision_processing
-import testing
-
-
-cameras = [
-    vision_processing.Camera(
-        (0.106, 0, 0.6606),
-        (0, -math.pi / 7.5),
-        650,
-        "testing/testing_resources/HyperClock Test Video.mp4"
-    )
-]
-object_detection = vision_processing.DynamicObjectProcessing()
-
-communications = testing.TestNetworkCommunication()
-
-while True:
-    dynamic_objects = []
-    reference_points = []
-    timestamp = time.time()
-
-    # Processing frames
-    for camera in cameras:
-        camera.update_frame()
-        dynamic_objects.extend(object_detection.get_dynamic_objects(camera))
-        reference_points.extend(vision_processing.ReferencePoint.from_apriltags(camera))
-
-    # Printing FPS
-    fps = 1 / (time.time() - timestamp)
-    print(f"Cycle was successful.\nFPS: {round(fps, 3)}")
+pipeline = PipelineRunner(test=True)
+pipeline.run()
```

Comment:
You don't have any assertions in this test. Is this for "smoke testing" manually by you?

Reply:
As for right now it is just a way for us to "smoke test" manually without having to run the code on the robot.